uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/directives/leave-request-popup.html

```diff
@@ -203,37 +96,15 @@ <h4 class="modal-title">Leave Request</h4>
                 </div>
               </div>
             </crm-loading>
-            <div class="row" ng-show="$ctrl.sickness">
+            <div class="row" ng-show="$ctrl.overtime">
               <div class="col-xs-12 form-horizontal">
-              <hr>
-                <div class="form-group">
-                  <label class="col-xs-4 control-label">Reason<i class="fa fa-asterisk"></i></label>
-                  <div class="col-xs-8">
-                    <div class="chr_custom-select chr_custom-select--full">
-                      <select name="reason">
-                        <option value="- select -">- select -</option>
-                      </select>
-                    </div>
-                  </div>
-                </div>
-                <div class="form-group">
-                  <div class="col-xs-12">
-                    <div>
-                      <input required type="checkbox" value="certification">
-                      <span>Self certification form required</span>
-                    </div>
-                    <div>
-                      <input required type="checkbox" value="interview">
-                      <span>Back to work interview required</span>
-                    </div>
-                    <div>
-                      <input required type="checkbox" value="work">
-                      <span>Return to work required</span>
-                    </div>
-                  </div>
-                </div>
+                <hr>
+                <div class="expire-date">Expiry Date: <span>{{$ctrl.currentDate()}}</div>
               </div>
             </div>
+            <div class="row" src="{{pathTpl + 'directives/sickness.html'}}" ng-show="$ctrl.sickness">
+              <ng-include src="pathTpl + 'request-modal/details/sickness.html'"></ng-include>
```

Comment:
I'm not sure if it would work, but couldn't you do simply
```html
<ng-include ng-show="$ctrl.sickness" src="pathTpl + 'request-modal/details/sickness.html'"></ng-include>
```
and get rid of that `div` wrapper? The `.row` element could be inside the included template